src/main.rs

```diff
@@ -15,24 +15,32 @@ fn send_usb_request(device: &rusb::Device<rusb::Context>) -> Result<()> {
         .filter(|l| l.lang_id() == MAGIC_LANGUAGE_ID)
         .for_each(|lang| {
             // Firmware call for Huion devices
-            let s = handle.read_string_descriptor(*lang, 201, timeout).unwrap();
-            println!("HUION_FIRMWARE_ID={s}");
+            let fwid = handle.read_string_descriptor(*lang, 201, timeout).unwrap();
+            /* firmware id is printed last, this messes up udev on some old devices */
```

Comment:
```suggestion
            // firmware id is printed last, this messes up udev on some old devices
```
for consistency


But really, if the *print* is failing and based on the commit message it's failing with some memory overwrite, then we need to figure out what's going on there...